web-frontend/src/pages/TransferCoins/ReceiveCoins/ReceiveCoins.tsx

```diff
@@ -14,6 +14,7 @@ function ReceiveCoins({ ...props }) {
   const presetURL = `${baseURL}${VITE_BASE_APP}/send-coins/?to=${
     account?.address || ''
   }`;
+  console.log(presetURL);
```

Comment:
```suggestion
```

Reply:
I thought I'd keep it for now to test the link easily

Reply:
Please, remove it!